modules/deployment-plug-in-cluster.adoc

```diff
@@ -6,36 +6,78 @@
 [id="deploy-on-cluster_{context}"]
 = Deploy your plug-in on a cluster
 
-After pushing an image with your changes to a registry, you can deploy the plug-in to a cluster.
+After pushing an image with your changes to a registry, you can deploy the plug-in to a cluster. A Helm chart is available to deploy a plug-in to a {product-title} environment. 
+
+.Prerequisites
+
+* You must have the Helm CLI installed. 
 
 .Procedure
 
-. To deploy your plug-in to a cluster, instantiate your plug-in by running the following command:
+. To deploy your plug-in to a cluster, install a Helm chart with the name of the plug-in as the Helm release name into a new namespace or an existing namespace as specified by the `-n` command-line option. Provide the location of the image within the `plugin.image` parameter by using the following command:
+
 +
 [source,terminal]
 ----
-$ oc process -f template.yaml \
-  -p PLUGIN_NAME=my-plugin \ <1>
-  -p NAMESPACE=my-plugin-namespace \ <2>
-  -p IMAGE=quay.io/my-repository/my-plugin:latest \ <3>
-  | oc create -f -
+$ helm upgrade -i  my-plugin charts/openshift-console-plugin -n my-plugin-namespace --create-namespace --set plugin.image=my-plugin-image-location
```

Comment:
Shall we clarify that https://github.com/openshift/console-plugin-template should be cloned first?

Reply:
it looks like we don't have content [console-plugin-template](https://github.com/openshift/console-plugin-template ) should be cloned first as an example in https://docs.openshift.com/container-platform/4.11/web_console/dynamic-plug-ins.html

Reply:
I have added a prerequisite step here. I am addressing adding steps to clone in [this topic](https://docs.openshift.com/container-platform/4.11/web_console/dynamic-plug-ins.html#getting-started-with-dynamic-plugins_dynamic-plugins) in a separate PR.